Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV SERVICE_TAGS=webapp \
     CONF_DIR=/usr/local/etc/redis \
     USER=redis \
     USER_ID=1000 \
-    REDIS_VERSION="6.2.12-r0" \
+    REDIS_VERSION="7.2.5-r0" \
```

Comment:
Is this major version upgrade planned? It's a heavy thing to roll this up during a very simple change.

I know Redis 6.2 sucks but for this change I'd rather do the light lifting and the major upgrade in another change when we know the consequences with consuming dogus

Reply:
Reduced to the latest available Redis version in the given alpine minor version